src/ovp_pipeline/commands/score_sources.py

```
@@ -0,0 +1,188 @@
+        print(f"vault not found: {vault}", file=sys.stderr)
+        return 2
+
+    providers = _build_providers(vault, domains_only=args.domains_only)
+    db_path = vault / "60-Logs" / "knowledge.db"
+    audit_path = vault / "60-Logs" / "source_authority.jsonl"
+
+    db_path.parent.mkdir(parents=True, exist_ok=True)
+    audit_path.parent.mkdir(parents=True, exist_ok=True)
+    conn = sqlite3.connect(db_path)
+    # Hold the audit log open for the duration of the batch — opening
+    # in append mode per record is ~2-3× slower at scale (444 sources
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)